src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+
+body {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 96px;
+  height: 116px;
+  margin: auto;
+}
+
+.stars {
+ display: flex;
+ flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
```

Comment:
its better to keep  background styles one by one:
background-image: url(./images/star.svg);
background-repeat: no-repeat;
background-position: center;

but the best option here is if you replace all of this like:

background: url(./images/star.svg) no-repeat center;